Documentation/network/servicemesh/envoy-circuit-breaker.rst

```diff
@@ -0,0 +1,250 @@
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/envoy/test-application-proxy-circuit-breaker.yaml
+
+The test workloads consist of:
+
+- one client deployments, ``fortio-deploy``
+- one service, ``echo-service``
+
+View information about these pods:
+
+.. code-block:: shell-session
+
+    $ kubectl get pods --show-labels -o wide
+    NAME                             READY   STATUS    RESTARTS   AGE     IP           NODE                       NOMINATED NODE   READINESS GATES   LABELS
+    echo-service-59557f5857-xh84s    2/2     Running   0          7m37s   10.0.0.125   cilium-control-plane   <none>           <none>            kind=echo,name=echo-service,other=echo,pod-template-hash=59557f5857
+    fortio-deploy-687945c6dc-6qnh4   1/1     Running   0          7m37s   10.0.0.109   cilium-control-plane   <none>           <none>            app=fortio,pod-template-hash=687945c6dc
+
+
+Configuring Envoy Circuit Breaker
+===========================================
+
+Apply the ``envoy-circuit-breaker.yaml`` file, which defines a ``CiliumClusterwideEnvoyConfig``.
+
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/envoy/envoy-circuit-breaker.yaml
+
+.. include:: warning.rst
+
+Verify the ``CiliumClusterwideEnvoyConfig`` was created correctly.
+
+.. code-block:: shell-session
+
+    $ kubectl get ccec envoy-circuit-breaker -oyaml
+    apiVersion: cilium.io/v2
+    kind: CiliumClusterwideEnvoyConfig
+    ...
+    resources:
+    - "@type": type.googleapis.com/envoy.config.cluster.v3.Cluster
+      name: "default/echo-service"
+      connect_timeout: 5s
+      lb_policy: ROUND_ROBIN
+      type: EDS
+      circuit_breakers:
+        thresholds:
+        - priority: "DEFAULT"
+          max_requests: 2
+          max_pending_requests: 1
+      outlier_detection:
+        split_external_local_origin_errors: true
+        consecutive_local_origin_failure: 2
+    services:
+    - name: echo-service
+      namespace: default
+
+In the ``CiliumClusterwideEnvoyConfig`` settings, we specified ``max_pending_requests: 1`` and ``max_requests: 2``. 
+These rules indicate that if we exceed more than one connection and request concurrently,
+We will see some failures when the envoy opens the circuit for further requests and connections.
+
+Tripping Envoy Circuit Breaker
+===========================================
+
+Make an environment variable with the pod ID for fortio:
+
+.. code-block:: shell-session
+
+    $ export FORTIO_POD=$(kubectl get pods -l app=fortio -o 'jsonpath={.items[0].metadata.name}')
+
+Call the service with two concurrent connections (-c 2) and send 20 requests (-n 20):
+
+.. code-block:: shell-session
+
+    $ kubectl exec "$FORTIO_POD" -c fortio -- /usr/bin/fortio load -c 2 -qps 0 -n 20 -loglevel Warning http://echo-service:8080
+
+Output::
+
+    $ kubectl exec "$FORTIO_POD" -c fortio -- /usr/bin/fortio load -c 2 -qps 0 -n 20 http://echo-service:808
```

Comment:
These two commands are different. Can you update them (and the output if needed) so they match?

Reply:
done.I update it to the same command.

Reply:
@learnitall I have updated the document for your all comment, please review ti again, thanks for your comment :).